.MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include README.md
```

Comment:
Python packages should have README in ReST format to include it as the long description for setup metadata.